BendAssist.App/BendAssists/CornerClose.cs

```diff
@@ -1,20 +1,108 @@
 ﻿using BendAssist.App.Model;
+using static BendAssist.App.Model.EOrientation;
+using BendAssist.App.Utils;
 
 namespace BendAssist.App.BendAssists;
 
 #region class CornerClose -------------------------------------------------------------------------
 public sealed class CornerClose : BendAssist {
    #region Constructors ---------------------------------------------
-   public CornerClose (Part part) => mPart = part;
+   public CornerClose (Part part) => (mPart, mIndex, mStepLines, mEdgeLines, mNewLines, mStepStartPts, mStepEndPts) = (part, -1, [], [], [], [], []);
    #endregion
 
    #region Methods --------------------------------------------------
    public override bool Assisted () {
       return true;
    }
+
+   /// <summary>Checks if a part is eligible for corner closing and gives the corner closed part</summary>
+   public override void Execute () {
+      base.Execute ();
+      var vertices = mPart!.Vertices; var pLines = mPart.PLines;
+      var commonVertices = vertices.GroupBy (p => p).Where (p => p.Count () > 2).Select (g => g.Key).ToList (); // To find the intersecting points of bendlines with stepcut 
+      if (commonVertices.Count < 1) mProcessedPart = null;
+      else {
+         var halfBA = GetBendAllowance (90, 0.38, 2, 2) / 2;

```

Comment:
Try to avoid magical numbers and give proper name to them and use them as variable.